tests/unit_tests/training/mimo/test_mimo_step.py

```
@@ -0,0 +1,163 @@
+        assert num_tokens.item() == 3
+        # Check metrics dict structure
+        assert 'lm loss' in metrics
+    
+    def test_loss_with_all_ones_mask(self):
+        """Test loss with all-ones mask."""
+        from megatron.bridge.training.mimo_step import loss_func
+        
+        output_tensor = torch.tensor([1.0, 2.0, 3.0])
+        loss_mask = torch.ones(3)
+        
+        total_loss, num_tokens, metrics = loss_func(loss_mask, output_tensor)
```

Ruff (0.14.14)

[warning] Unpacked variable `metrics` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)